book/src/configuration.md

```diff
@@ -169,6 +169,7 @@ All git related options are only enabled in a git repository.
 |`git-ignore` | Enables reading `.gitignore` files | true
 |`git-global` | Enables reading global `.gitignore`, whose path is specified in git's config: `core.excludefile` option | true
 |`git-exclude` | Enables reading `.git/info/exclude` files | true
+|`require-git` | Enables use of local and global .gitignore files even if not in a git directory (if set to `false`) | true
```

Comment:
I think it's a but odd to describe a setting by it's inverse (si what disableing it changes). We usually just describe what the flag enable and then show the default aftwards. For example:


```suggestion
|`require-git` | Only use (local and global) .gitignore if inside a git repository/worktree | true
```

Reply:
Ah OK, I was trying to keep the same phrasing of "Enables..." but this does make more sense. Will update this and comment.